database/gormdb_assignment.go

```diff
@@ -69,7 +68,9 @@ func (db *GormDB) GetAssignmentsByCourse(courseID uint64, withGrading bool) ([]*
 	if withGrading {
```

Comment:
Do we have a test for `withGrading == true`?

Reply:
Looks like we don't. Will add one.

Reply:
Done.